### ISSUE TITLE
media-sound/cadence: Removal?

### ISSUE THREAD

**Issue by NexAdn** [gentoo-audio/audio-overlay#513]
media-sound/cadence has both a versioned and live ebuild in ::gentoo. As such, having a live ebuild of our own is a bit redundant.

I propose we remove media-sound/cadence from ::audio-overlay.

**Comment by simonvanderveldt** [comment]
AFAIK the one in ::gentoo doesn't have LADISH support. I tried NSM some time ago, but the UX was significantly worse compared to LADISH/Claudia.

**Comment by NexAdn** [comment]
Just found [Bug 442916](https://bugs.gentoo.org/442916) for adding LADISH to ::gentoo. Doesn't look like they are against adding it, so we might be able to get it into ::gentoo.
But I then it's left to decide whether to maintain another ebuild in ::gentoo or keep the current media-sound/cadence in ::audio-overlay.

**Comment by simonvanderveldt** [comment]
Yeah, there's a now decade old :O bug for it https://bugs.gentoo.org/442916
Assuming we can exclude the GTK2 frontend (gladish) the only missing thing would be a new release because the latest existing one (https://github.com/LADI/ladish/tree/ladish-1) is not compatible with Python3. Nedko's availability/reachability is kinda spotty though, so not sure if we can make that happen.
